fanficfare/adapters/adapter_readonlymindcom.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+# -- coding: utf-8 --
+# Copyright 2013 Fanficdownloader team, 2021 FanFicFare team
```

Comment:
Is there a reason for the two different coding comments?
Copyright on a new file should only be this year.

Reply:
That's just what was in the mcstories adapter. I can remove the old one.